test/python/transpiler/test_swap_strategy_router.py

```diff
@@ -77,7 +78,13 @@ def test_basic_zz(self):
 
         """
 
-        op = PauliSumOp.from_list([("IZZI", 1), ("ZIIZ", 2), ("ZIZI", 3)])
+        with warnings.catch_warnings(record=True) as caught_warnings:
+            warnings.filterwarnings(
+                "always",
+                category=DeprecationWarning,
+            )
+            op = PauliSumOp.from_list([("IZZI", 1), ("ZIIZ", 2), ("ZIZI", 3)])
```

Comment:
Once https://github.com/Qiskit/qiskit-terra/pull/9537 is merged, we can update this test not to use opflow.